### PR TITLE
Initialize library

### PR DESCRIPTION
**Pull request by ashton-herrington**

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1 @@
+* @moovfinancial/cards
```
Comment (ashton-herrington) [inline .github/CODEOWNERS:1]: I assume for public repos this needs to be a user not internal group, will fix

File: .github/workflows/cgo.yml [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    name: Go Build (CGO)
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest, macos-latest, windows-latest]
+    steps:
+    - name: Set up Go 1.x
+      uses: actions/setup-go@v2
+      with:
+        go-version: ^1.16
```
Comment (adamdecaf) [inline .github/workflows/cgo.yml:20]:
```suggestion
        go-version: ^1.19
```

**Comment by gyrolc** [comment]
Looking good. Should this also include a FIPS to alpha state code converter?

**Comment by ashton-herrington** [comment]
Good idea, will add

File: code_mapper.go [hunk 3]
```diff
@@ -0,0 +1,154 @@
...
+// specific language governing permissions and limitations
+// under the License.
+
+package fips_state_codes
+
+import (
+	"fmt"
+)
+
+var stateCodeToVIPSCodeMap = map[string]string{
```
[the remark this answers is not in the document]
Comment (ashton-herrington) [reply code_mapper.go:24]: Dude.....it should!

**Review by gyrolc — APPROVED** [review]

File: .github/workflows/nocgo.yml [hunk 4]
```diff
@@ -0,0 +1,37 @@
+name: Go
```
Comment (alovak) [inline .github/workflows/nocgo.yml:1]: I would just keep only one file `go.yml` as we don't build any binary and there is no difference `cgo` or `nocgo`

**Pull request merged**